### PR TITLE
Fixes #254 + discrepancies of AdaBoostRegressor

### LINKED ISSUE #254: Adaboost classifier and regressor throw error with non-default base_estimator hyper-parameter
data = load_iris()
X, y = data.data, data.target
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.5, random_state=42)

model = AdaBoostClassifier(base_estimator=RandomForestClassifier())
model.fit(X_train, y_train)

onnx_fs = convert_sklearn(model, 'ada', [('float_input', FloatTensorType([1, X_test.shape[1]]))])
save_model(onnx_fs, 'ada_cla.onnx')
sess = InferenceSession('ada_cla.onnx')
res = sess.run('', input_feed={'float_input': X_test.astype('float32')})
print(np.mean(np.isclose(model.predict_proba(X_test), list(map(lambda x:
                                                         list(map(lambda y: x[y], x)), res[1])), atol=1e-7)))
print(np.mean(res[0] == np.argmax(model.predict_proba(X_test), axis=1)))

~/Documents/MachineLearning/onnx_projects/onnx_env/lib/python3.6/site-packages/skl2onnx-1.5.1-py3.6.egg/skl2onnx/operator_converters/ada_boost.py in convert_sklearn_ada_boost_classifier(scope, operator, container)
    160             attrs['classlabels_strings'] = classes
    161 
--> 162         add_tree_to_attribute_pairs(attrs, True, op.estimators_[tree_id].tree_,
    163                                     0, 1, 0, True, True, dtype=container.dtype)
    164         container.add_node(

AttributeError: 'RandomForestClassifier' object has no attribute 'tree_'

data = load_diabetes()
X = data.data
y = data.target

X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.5, random_state=42)
model = AdaBoostRegressor(base_estimator=LinearRegression(), n_estimators=5, learning_rate=1,
loss='exponential', random_state=42).fit(X_train, y_train)

onnx_model = convert_sklearn(model, 'lr', [('input', FloatTensorType([1, X_test.shape[1]]))])
save_model(onnx_model, 'lr.onnx')
sess = InferenceSession('lr.onnx')
res = []
for x in X_test:
    res.append(sess.run(None, input_feed={'input': x.astype(np.float32)}))

print(np.mean(np.isclose(model.predict(X_test), list(map(lambda x: x[0][0][0], res)), rtol=1e-5)))

~/Documents/MachineLearning/onnx_projects/onnx_env/lib/python3.6/site-packages/skl2onnx-1.5.1-py3.6.egg/skl2onnx/operator_converters/ada_boost.py in _get_estimators_label(scope, operator, container, model)
    232         attrs = get_default_tree_regressor_attribute_pairs()
    233         attrs['name'] = scope.get_unique_operator_name(op_type)
--> 234         attrs['n_targets'] = int(model.estimators_[tree_id].n_outputs_)
    235         add_tree_to_attribute_pairs(attrs, False,
    236                                     model.estimators_[tree_id].tree_,

AttributeError: 'LinearRegression' object has no attribute 'n_outputs_'

### PR DESCRIPTION
**Pull request by xadupre**
Does not fix issue #254.

### CODE REVIEW

File: skl2onnx/operator_converters/ada_boost.py [hunk 1]
```diff
@@ -356,13 +356,13 @@ def convert_sklearn_ada_boost_regressor(scope, operator, container):
                        median_idx_name,
                        name=scope.get_unique_operator_name('ArgMin'), axis=1)
     container.add_node(
-        'ArrayFeatureExtractor', [sorted_indices_name, median_idx_name],
-        median_estimators_name, op_domain='ai.onnx.ml',
-        name=scope.get_unique_operator_name('ArrayFeatureExtractor'))
+        'GatherElements', [sorted_indices_name, median_idx_name],
```
Comment (prabhat00155) [inline skl2onnx/operator_converters/ada_boost.py]: What does GatherElements do? Is this a newly added op? I don't see it here: https://github.com/onnx/onnx/blob/master/docs/Operators-ml.md

Comment (xadupre) [reply skl2onnx/operator_converters/ada_boost.py]: Yes it is here: https://github.com/onnx/onnx/blob/master/docs/Operators.md#GatherElements

cumsum was added too so I'll use it too.

Comment (prabhat00155) [reply skl2onnx/operator_converters/ada_boost.py]: That means it's available in version 11 of the default onnx operator set. You should add version=11 and remove op_domain(which is wrongly set to ai.onnx.ml). Were you able to test this?

Comment (xadupre) [reply skl2onnx/operator_converters/ada_boost.py]: Fixed. I implemented a version of gatherelements with other ONNX operators.

**Pull request merged**